Backpack/build.gradle

```diff
@@ -60,12 +60,13 @@ android {
 
 dependencies {
   api rootProject.ext.androidxAnnotations
+  api "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
```

Comment:
to match everything else can we define this in the `ext` part as well?

Reply:
Nope – since it's used in classpath dep[endencies of top-level buildscript it fails.